platform_versioned_docs/version-24.2/enterprise/advanced-topics/manual-azure-batch-setup.mdx

```diff
@@ -0,0 +1,487 @@
+
+### 8. Use a node pool attached to a VNet with worker nodes attached to the same VNet
+
+Finally, you can combine some of the previous approaches. Nextflow can create and modify Azure Batch pools based on the pipeline requirements. You can also attach Azure Batch pools to a VNet. Next, attach the worker nodes to the same VNet. 
+
+To achieve this, the following requirements must be met:
+
+- The pipeline must be launched on the node pool attached to the VNet.
+- The managed identity must be used to authenticate to Azure Batch and Storage.
+- The managed identity must have permissions to create resources attached to the VNet.
+- Nextflow creates node pools attached to the VNet. 
+
+Do the following:
+
+1. Duplicate the `hello-world-entra-mi` pipeline, but modfiy the compute environment to `3-azure-batch-vnet` and change the pipeline name to `hello-world-vnet`.
+1. Check the virtual network string under the pool details in the Azure Portal, under the **Network Configuration** section. The value should be a Subnet ID, such as `/subscriptions/${subscriptionId}/resourceGroups/${resourceGroup}/providers/Microsoft.Network/virtualNetworks/${vnetName}/subnets/${subnetName}`. Save this value.
+1. Change the Nextflow configuration under the **Advanced** tab to include a virtual network with the autopools:
+
+```nextflow
+process.queue = "auto"
+process.machineType = "Standard_E*d_v5"
+azure {
+    batch {
+        autoPoolMode = true
+        allowPoolCreation = true
+        pools {
+            auto {
+                autoscale = true
+                vmCount = 1
+                maxVmCount = 4
+                virtualNetwork = '/subscriptions/${subscriptionId}/resourceGroups/${resourceGroup}/providers/Microsoft.Network/virtualNetworks/${vnetName}/subnets/${subnetName}'
+            }
+        }
+    }
+}
+```
+
+Select **Launch** next to the `hello-world-vnet` pipeline in your workspace Launchpad to complete the launch form and launch the workflow.
+
+The pipeline runs as before, but using the managed identity to authenticate to Azure Batch and Storage. It also creates worker pools attached to the VNet.
+
+Behavior:
+
+- The same as the previous example, but with additional worker node pools attached to the VNet.
+
+Advantages:
+
+- The same as the previous example, with the additional flexibility of different node sizes per task requirements.
+
+Disadvantages:
+
+- The same as before, but with increased complexity which can introduce errors.
+
+### 9. Clear up resources
+
+Once you have completed setup and workflow execution, you can delete the pipelines and compute environments from Seqera.
+
+In Azure, you can delete the Batch account, which will delete all pools, jobs, and tasks. You can then delete the Storage account.
+
+If you wish to keep the Azure resources, you can remove each pool within a Batch account and mark any active jobs as terminated.
+
+## Conclusion
```

Comment:
@adamrtalbot Per my Slack, if we cut out the conclusion section for now, we should be able to publish what we have and you can add expand on this further when you're ready for it. LMK